src/github3/events.py

```diff
@@ -438,7 +438,8 @@ def _update_attributes(self, event):
         # If we don't copy this, then we end up altering _json_data which we do
         # not want to do:
         event = copy.deepcopy(event)
-        self.actor = EventUser(event["actor"], self)
+        if event["actor"]:
+            self.actor = EventUser(event["actor"], self)
```

Comment:
I mean people will access `event.actor` and get an Attribute Error

Reply:
Gotcha. So you're suggesting to assign`None` explicitly?

```suggestion
            self.actor = EventUser(event["actor"], self)
        else:
            self.actor = None
```

PS: I had no problems checking `event.actor` as follows:

```
if event.actor:
    user = event.actor.login
else:
    user = "ghost"
```